README.md

```diff
@@ -55,10 +55,10 @@ Installing from source is only recommended if you are comfortable using the comm
 #### Mautic requirements
 
 1. See [Mautic requirements](https://www.mautic.org/download/requirements).
-2. PHP modules:
-	- required: `zip`, `xml`, `mcrypt`, `imap`, `mailparse`
-	- recommended: `openssl`, `opcache` / `apcu` / `memcached`
-	- recommended for development: `xdebug`
+2. PHP extensions:
+	- required: `zip`, `xml`, `openssl`, `mailparse`, `json`, `session`, `hash`, `iconv`, `ctype`, `mysqli`, `mbstring`, `libxml`, `SimpleXML`, `curl`, `filter`, `fileinfo`, `posix`, `gd`, `bcmath`, `intl`, `readline`
```

Comment:
Not sure about `openssl` here. I think this could be BC, because we introduced openssl support in 2.12 and older installation could run on PHP without this feature.
What do you think?

Reply:
Good point. I guess I could make a note about mcrypt and why people should not use it anymore.